student-store-ui/src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react"
+import "./Hero.css"
```

Comment:
If you are going to add the imports, I would set up the css files too. Your compiler might complain/crash if it cannot find them. They can be empty.